app/helpers/form_helper.rb

```diff
@@ -66,4 +66,9 @@ def us_states_territories
       ['Wyoming', 'WY'],
     ]
   end
+
+  # Filtered list to removve territories that error due to USPS API
+  def filtered_us_states_territories
```

Comment:
Can we move this to a class/module more specific to USPS, or at the very least prefix it with something indicating that it's USPS-related. This module is a global helper, so "filtered" in this context leaves a lot of room for ambiguity.

Reply:
Sure, would a file like app/helpers/ipp_form_helper.rb do the trick? Or should it live elsewhere entirely to be more context specific?

Reply:
Technically that'd work, but helpers are global methods. That makes it convenient, but it comes with [all the downsides of global variables](https://embeddedartistry.com/fieldatlas/the-problems-with-global-variables/). I think the idea with `us_states_territories` is that it's a pretty common use-case for wanting all U.S. states, so it's more reasonable to have globally available. For something like in-person proofing, I'd probably suggest a [presenter](https://github.com/jwipeout/rails-directory-structure-guide?tab=readme-ov-file#-apppresenters), or an instance variable assigned in the controller action, or even just performing the filtering within the view itself.

Reply:
Sweet, thanks! I'll look into presenters since I think some abstraction would be helpful as the list is used across a few different views.

Reply:
Have we considered adding an option to `us_states_territories`? We could make it optional to simplify existing code, or make it required to make it clear? This would remove the need for an extra helper class, etc

```ruby
def us_states_territories(usps_states_only: false)
  [
    # ...
  ].reject { |_name, abbrev| usps_states_only && %w[AA AE AP UM].include?(abbrev) }
end
```

Reply:
I like that idea personally... After looking into presenters it seems a little over kill for this use case? Plus, if the list needs to be in both controllers and views presenter maybe feels weird...

Reply:
@aduth You think Zachs idea above is acceptable? Want to reach consensus if we can on what solution this needs. 

(P.S. looks like this list  actually is only used in views, so Presenter could make more sense than I was originally thinking, but still seems maybe a bit intense for this.)

Reply:
I'm not a fan of letting IPP-specific considerations bleed into this global application helper, but it's not a hill I'd die on.

Reply:
Makes sense! Ill just go presenter here then